core/lib/libmbedtls/sub.mk

```diff
@@ -0,0 +1,3 @@
+cflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
+
+subdirs-y += src
```

Comment:
I'd prefer if we put everything below `lib/libmbedtls` instead, having two different `libmbedtls` directories will be confusing.

The `src` directory will be `core` only we, so we could rename it to `core` and make this line:
```
subdirs-$(sm-core) += core
```

Reply:
Yes, it will be better to remove that confuse. I will update as that.